packages/cli/src/commands/generate/generator/generator.js

```diff
@@ -0,0 +1,97 @@
+import path from 'path'
+
+import fse from 'fs-extra'
+import Listr from 'listr'
+import terminalLink from 'terminal-link'
+
+import { getPaths } from '../../../lib'
+import c from '../../../lib/colors'
+
+export const command = 'generator <name>'
+export const description =
+  'Copies generator templates locally for customization'
+
+const SIDE_MAP = {
+  web: ['cell', 'component', 'layout', 'page', 'scaffold'],
+  api: ['function', 'sdl', 'service'],
+}
+
+const copyGenerator = (name, { force }) => {
```

Comment:
Crazy idea 3: is there any way we could add to the top of a custom template (maybe something like front matter) what variables are available to use for that generator?

Reply:
Hmmm...so there would be some kind of "optional" comment in the template that gets stripped out if you generate from the default template, but would still be present if you copy the template down to your local app...hmmmmmmmmm. I'll look at that.......

Reply:
Oh man as I'm looking into these there's a bunch of unique ones across different generators. I'm really not looking forward into figuring out what these all are. haha 

I figured people that are customizing can figure them out from their names and the context they're used in in the templates. I'm guessing people are going to be making more HTML/CSS changes to the templates to match their app's look and feel, not so much creating whole new functionality based on the variables in here.

Let's launch like this and see if people complain!